pandas/tests/util/test_show_versions.py

```diff
@@ -26,11 +37,44 @@
     "ignore:Distutils:UserWarning"
 )
 @pytest.mark.filterwarnings("ignore:Setuptools is replacing distutils:UserWarning")
-def test_show_versions(capsys):
+def test_show_versions(tmpdir):
```

Comment:
below every test, can you add a comment with the issue number? something like
```
# GH39701
```
, see other recent tests